### DISCUSSION TITLE
Can we get a Rails 6 guide?

### DISCUSSION THREAD

**Discussion by joooooohnx** [johncorderox/Rails-API-React-Tutorial#16] [Q&A] [answered]
Rails 6 is brand new! Im willing to contribute and help for rails 6 apis and updating the template. Maybe we can have 1 rails 5 api folder and a rails 6 api one too 🤔

**Comment by johncorderox** [comment, marked as answer]
I also think this is essential since rails APIs are skyrocketing for startups. I'm a bit busy but we can start scaffolding really soon.

**Reply by joooooohnx** [reply]
Instead of using the react gem should I just use create-react-app?

**Reply by johncorderox** [reply]
perfect bud

**Reply by johncorderox** [reply]
its done now :)
